test-app/package.json

```diff
@@ -19,6 +19,7 @@
     "aws-cdk": "2.20.0",
     "constructs": "10.0.0",
     "functionless": "file:../",
+    "@functionless/language-service": "file:../../functionless-language-service",
```

Comment:
Removed for now because i can't release language-service until the validate function exists in functionless. Will first merge and release this, then i will release language-service, then come back and add it to test-app with another release.